llama_index/chat_engine/condense_question.py

```diff
@@ -178,6 +186,14 @@ def stream_chat(
         if self._verbose:
             print(log_str)
 
+        # TODO: right now, query engine uses class attribute to configure streaming,
+        #       we are moving towards separate streaming and non-streaming methods.
+        #       In the meanwhile, use this hack to toggle streaming.
+        from llama_index.query_engine.retriever_query_engine import RetrieverQueryEngine
+
+        if isinstance(self._query_engine, RetrieverQueryEngine):
+            self._query_engine._response_synthesizer._streaming = True
```

Comment:
are we going to set it back after the query call is run?

Reply:
good point, sure.